sql_joins/sql_joins.md

```diff
@@ -0,0 +1,875 @@
+<!--
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+module_template_version: 3.0.0
+language: en
+narrator: US English Female
+title: SQL Joins
+comment: Learn about SQL joins: what they accomplish, and how to write them.
+long_description: Usually, data in a SQL database is organized into multiple interrelated tables.  This means you will often have to bring data together from two or more tables into a single dataset to answer your research questions.  This "join" action is accomplished using `JOIN` commands.  This module teaches types of joins, join criteria, and how to write `JOIN` code.
+estimated_time: 1 hour
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
```

Comment:
Suggesting on multiple lines doesn't work so well... but I suggest reordering the objectives to match the order of the module:
- Understand the parts of a JOIN
- Describe the "shapes" of SQL JOINs: inner, left, right, and full
- Explain what "join criteria" are